src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -2807,4 +2807,4 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\Wasi\WasiPollWorld.wit.imports.wasi.io.v0_2_1.IPoll.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Threading\Wasi\WasiPollWorld.wit.imports.wasi.io.v0_2_1.PollInterop.cs" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
I'm not sure that whitespace-only difference is worth it. There 3 such changes.

Reply:
Sorry, those are artifacts of conflict resolution.  I'll remove them.